test/test_tree_2d_shallowwater.jl

```diff
@@ -222,7 +222,7 @@ end
     end
 end
 
-@trixi_testset "elixir_shallowwater_source_terms.jl with flux_hll" begin
+@trixi_testset "elixir_shallowwater_source_terms.jl with FluxHLL(min_max_speed_naive)" begin
```

Comment:
I'm not sure whether we really want to test the naive version here or the Davis version.